components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -4296,6 +4296,36 @@ def searchProcessing():
             search.close()
         return rv
 
+    def getThumbnailSet(self, image_ids, max_size=64):
```

Comment:
I would not conflate BlitzGateway defaults with OMERO.web defaults. While they are obviously quite related, and 5.3.0 is a breaking release, changing the BlitzGateway default behaviour has potentially far reaching consequences. We are unlikely to be able to fully explore them this late in the 5.3.0 development cycle.